hackathon_site/event/jinja2/event/landing.html

```diff
@@ -207,23 +207,21 @@ <h3 class="center white-text">FAQ</h3>
                         </li>
                         <li>
                             <div class="collapsible-header">
-                                What equipment and support will be provided?
+                                What software/hardware can we use?
                             </div>
                             <div class="collapsible-body">
                                 <span>
-                                    Unfortunately, due to the current situation of COVID19, we won’t be able to provide any hardware. However, there will be lots of mentors ready to help you! During the online
-                                    hackathon, there will be a breakout room for all the mentors where you will be able to access at all times and get immediate help.
+                                As this hackathon is online, you can use any of the software or hardware that you have available at your house.
                                 </span>
                             </div>
                         </li>
                         <li>
                             <div class="collapsible-header">
-                                How do online hackathons work?
+                                Is this hackathon online?
                             </div>
                             <div class="collapsible-body">
                                 <span>
-                                    The entire hackathon will be conducted online. You will be able to stay safe and comfortably at home while doing this hackathon. You will be able to communicate to your team via
-                                    online meetings and you will be submitting your final products online.
+                                    Yes, this hackathon will be hosted completely online using multiple online communication platforms. So you can enjoy the hacking experience from the comfort of your home.
```

Comment:
Minor gramatical change: 
```suggestion
Yes, this hackathon will be hosted completely online using multiple online communication platforms so you can enjoy the hacking experience from the comfort of your home.
```